source/renderer/app/components/wallet/settings/ChangeSpendingPasswordDialog.js

```diff
@@ -127,6 +140,9 @@ export default class ChangeSpendingPasswordDialog extends Component<Props> {
           validators: [
             ({ field, form }) => {
               const repeatPasswordField = form.$('repeatPassword');
+              if (repeatPasswordField.value.length === 0) {
+                repeatPasswordField.validate({ showErrors: false });
+              }
               if (repeatPasswordField.value.length > 0) {
                 repeatPasswordField.validate({ showErrors: true });
               }
```

Comment:
@DeeJayElly Same thing with isolating clauses as in above comment

Reply:
Fixed